### PR TITLE
Change: Do not sleep on exclusive zones

### PR DESCRIPTION
**Pull request by acasajus**

### CODE REVIEW

File: app/parallel_limiter.py [hunk 1]
```diff
@@ -22,24 +21,16 @@ def __init__(
         lock_suffix: Optional[str] = None,
         max_wait_secs: int = 5,
```
Comment (nguyenkims) [inline app/parallel_limiter.py:22]: we can also remove `max_wait_secs` right?

File: tests/test_prarallel_limiter.py [hunk 2]
```diff
@@ -8,6 +8,6 @@ def test_parallel_limiter(flask_app):
     user = create_new_user()
     with flask_app.test_request_context():
         login_user(user)
-        pl = _InnerLock("test", max_wait_secs=1, wait_loop_secs=0.1)
+        pl = _InnerLock("test", max_wait_secs=1)
```
Comment (nguyenkims) [inline tests/test_prarallel_limiter.py:11]: can we test a case that raises the TooManyRequest exception? For example by having a slow operation in the wrapped function.

File: app/parallel_limiter.py (hunk 1 above)
Comment (acasajus) [reply app/parallel_limiter.py:22]: That defines what's the max time redis will hold the lock.

File: tests/test_prarallel_limiter.py (hunk 2 above)
Comment (acasajus) [reply tests/test_prarallel_limiter.py:11]: I tried to but didn't manage to have two parallel threads running with pytest. It seems to make threads work serially. Either that or `time.sleep` doesn't release the GIL.

**Pull request merged**